sass/directives/_tables.scss

```diff
@@ -40,64 +40,19 @@
 
 
 
-@mixin table--row-column-headers($table-highlight-z-index: 1, $table-caption-z-index: 2) {
-  border-collapse: separate;
-  border-spacing: $table-border-spacing 0;
-  table-layout: fixed;
-  width: 100%;
+@mixin table--row-column-headers($table-highlight-z-index: 1, $table-caption-z-index: 2, $table-col-width: 25%) {
   overflow: hidden;
```

Comment:
Didn't you need to remove `overflow: hidden` for your "Most Popular" flag to work? If so I'd move `overflow` to the `table__column-highlight` mixin and set that up to be applied to the `table` and not the `td` and `th`s, if not leave it as is